cli/sidecar.py

```diff
@@ -0,0 +1,9 @@
+import sys, importlib.util as iu
+spec = iu.spec_from_file_location("cyclus.system", "../cyclus/system.py")
+mod = iu.module_from_spec(spec)
+sys.modules["cyclus.system"] = mod
+spec.loader.exec_module(mod)
```

Comment:
Is this all necessary because we want to use this module before it is installed in the python path, i.e. during build time?

Reply:
Yes, it is difficult to import `cyclus.system` in the `#pragma cyclus exec`s during build time, thus the importing via filepath.  I just went back through to try to find a better solution and was unsuccessful